packages/nextjs/app/page.tsx

```diff
@@ -15,8 +15,12 @@ const Home: NextPage = () => {
           </h1>
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
           <p className="text-lg flex gap-2 justify-center">
-            <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+            <span className="font-bold">
+              Checked-in builders{" "}
+              <a href="/builders" className="link">
```

Comment:
You might want to replace the `<a>` tag with a `<Link>` component from Next.js. This is the preferred way to create internal links in Next.js applications, as it provides better performance and client-side navigation.